libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -59,4 +63,16 @@ export class BookSearchComponent implements OnInit {
       this.store.dispatch(clearSearch());
     }
   }
+
+  openSnackBar(message: string, action: string, book: Book) {
+    const snackBarRef = this._snackBar.open(message, action, {
+      duration: 3500
```

Comment:
Is that long enough for a partially sighted user to find the snackbar?

Reply:
updated